sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -117,25 +117,42 @@ should be used.
 To run SYCL  applications on OpenCL devices, OpenCL implementation(s) must be
 present in the system.
 
-To execute SYCL tests on Intel devices following runtimes should be installed:
+The following `GPU` runtimes should be installed to run SYCL applications on  Intel `GPU` devices:
```

Comment:
Please, limit line length to 80 characters.

Reply:
Fixed.